test/aws_secrets_provider/start

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -ex
+
+./stop
```

Comment:
This isn't consistent with other test scripts. Start should bring everything online and then test would run the test code rather than test running it all.